aws/logs_monitoring/lambda_function.py

```diff
@@ -461,6 +477,17 @@ def generate_metadata(context):
     return metadata
 
 
+def extract_trace(event):
+    """Extract traces from an event if possible"""
+    try:
+        message = event["message"]
+        if not trace_regex.match(message):
```

Comment:
Can't we use something else than a regex to match? Aka reuse the event field maybe ?

Reply:
if the schema is static, I think it's probably better to check the existence of the attribute in the map, WDYT ?

Reply:
Thanks. I've removed the regex, and parsed the line as json instead to check for the field.